source/_integrations/rako.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Rako Controls Lights
+description: Instructions on how to integrate Rako Controls Lights with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2021.5.0
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@marengaz'
+ha_domain: rako
+---
+
+
+Rako smart lighting controls lead the way in providing state-of-the-art digital dimming technology, providing innovative solutions to meet the needs of a diverse range of applications.
+
+The [Rako Controls Hub](https://rakocontrols.com/wkhub/) and its predecessor, the Rako Controls Bridge module, offer local network control of Rako's lighting modules. The Bridge has since been discontinued in favour of the Hub.
```

Comment:
```suggestion
The [Rako Controls Hub](https://rakocontrols.com/wkhub/) and its predecessor, the Rako Controls Bridge module, offer local network control of Rako's lighting modules. The Bridge has since been discontinued in favor of the Hub.
```